website/layouts/shortcodes/heronApiVersion.ace

```diff
@@ -0,0 +1 @@
+{{.Page.Site.Params.versions.heronapi}}
```

Comment:
We should have only one heron version. And api, spi, cli or others should use that version number too. It makes all sub-components in heron with consistent version.